RUFAS/routines/manure/pen_manure/manure_manager_pen.py

```diff
@@ -155,6 +151,10 @@ def barn_area_from_pen_type(self) -> float:
 
         barn_area = barn_area_by_pen_type[self.pen_type]
 
-        if "Cow" in self.classes_in_pen:
+        if (
+            self.animal_combination == AnimalCombination.LAC_COW
+            or self.animal_combination == AnimalCombination.GROWING_AND_CLOSE_UP
+            or self.animal_combination == AnimalCombination.CLOSE_UP
+        ):
```

Comment:
```suggestion
        pen_contains_cows = (
            self.animal_combination == AnimalCombination.LAC_COW
            or self.animal_combination == AnimalCombination.GROWING_AND_CLOSE_UP
            or self.animal_combination == AnimalCombination.CLOSE_UP
        )
        if pen_contains_cows:
```
When writing if statements, it is good to make the conditional its' own variable if it is not trivial